.github/workflows/ci.yml

```diff
@@ -0,0 +1,72 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+    tags: ['*']
+    paths-ignore:
+      - 'CITATION.bib'
+      - 'LICENSE.md'
+      - 'README.md'
+      - '.zenodo.json'
+      - '.github/workflows/CompatHelper.yml'
+      - '.github/workflows/SpellCheck.yml'
+      - '.github/workflows/TagBot.yml'
+      - 'docs/**'
+  pull_request:
+    paths-ignore:
+      - 'CITATION.bib'
+      - 'LICENSE.md'
+      - 'README.md'
+      - '.zenodo.json'
+      - '.github/workflows/CompatHelper.yml'
+      - '.github/workflows/SpellCheck.yml'
+      - '.github/workflows/TagBot.yml'
+      - 'docs/**'
+  workflow_dispatch:
+
+concurrency:
+  # Skip intermediate builds: always.
+  # Cancel intermediate builds: only if it is a pull request build.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+
+jobs:
+  test:
+    name: ${{ matrix.os }} - Julia ${{ matrix.version }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.9'
```

Comment:
Shall we just use Julia v1.9 or also v1.8 as we do in Trixi.jl?

Reply:
Right!
```suggestion
          - '1.8'
          - '1.9'
```

Reply:
Why not test with the latest release as well?

Reply:
Yes yes...

Reply:
Now, we have to update this with every version. Why don't we just add the version `1`? Or `latest`?

Reply:
Well, if we officially support Julia v1.8 through latest with Trixi.jl, we should also test for it. One could argue that the tests here would suffice to be just very basic tests on one Julia version and OS, but OTOH, since the tests will be very fast, it does not hurt to run them all...

Reply:
I mean instead of `1.10` or so.

Reply:
v1.8 was the last version without pkgimages, v1.9 the first one with. For me, 1.8, 1.9 and latest would thus be ok.